README.md

```diff
@@ -1,209 +1,26 @@
-[![instantsearch.js logo][readme-logo]][website-url]
+[![InstantSearch.js logo][readme-logo]][website]
 
-[![Version][version-svg]][package-url] [![Build Status][travis-svg]][travis-url] [![License][license-image]][license-url] [![Downloads][downloads-image]][downloads-url]
+InstantSearch projects: **InstantSearch.js** [React InstantSearch][react-instantsearch-github], [InstantSearch Android](instantsearch-android-github).
 
-**instantsearch.js** is a library of UI widgets to help you build the best instant-search experience with [Algolia's Hosted Search API](https://www.algolia.com/?utm_medium=social-owned&utm_source=GitHub&utm_campaign=InstantSearch%20repository).
+## InstantSearch.js
 
-Have a look at the website: [https://community.algolia.com/instantsearch.js/][instantsearch-website].
+> ⚡ Lightning-fast search for websites
 
```

Comment:
maybe a small link to `react-instantsearch` or to the upcoming landing page for instantsearch here?